config/webpack.dev.config.js

```diff
@@ -62,7 +44,8 @@ module.exports = Merge.smart(commonConfig, {
               plugins: () => [
                 /* eslint-disable global-require */
                 require('autoprefixer'),
-                require('postcss-prepend-selector')({ selector: '.SFE ' }),
+                require('postcss-initial')(),
+                require('postcss-prepend-selector')({ selector: '#root.SFE ' }),
```

Comment:
I had to make this selector more specific because there are some legacy Studio selectors that were still getting past my `all: initial` wall because they were overly specific. It's possible there are even more specific selectors that this doesn't override, but there at least aren't any on the Files & Uploads page.